numpy/lib/function_base.py

```diff
@@ -338,6 +338,10 @@ def histogram(a, bins=10, range=None, normed=False, weights=None,
         if (mn > mx):
             raise AttributeError(
                 'max must be larger than min in range parameter.')
+        if not np.all(np.isfinite([mn, mx])):
+            raise AttributeError(
```

Comment:
I suppose you have tried to replicate what was already there, but this should be a `ValueError`, since the problem is not that ["an attribute reference or assignment fails,"](https://docs.python.org/3.5/library/exceptions.html#AttributeError) but ["an inappropriate value."](https://docs.python.org/3.5/library/exceptions.html#ValueError)

You don't have to change it in this PR, but I think the `AttributeError` on line 339 should be changed to a `ValueError` as well.